channel/item.go

```diff
@@ -391,7 +391,8 @@ func (v item) bytes(shortSlot bool) []byte {
 		p.WriteUint64(uint64(v.id)) // I think this is somekind of cashshop transaction ID for the item
 	}
 
-	p.WriteInt64(v.expireTime)
+	//p.WriteInt64(v.expireTime)
```

Comment:
Keep it as the variable and make the default in the item creation the value below.

Reply:
changed it to a const and updated so that all items have it by default. Still need to figure out the saving issue.

Reply:
I might not be a saving issue. The way items are handled when moved/dropped might need to be refactored and simplified as I think that might be what is causing the state of the server and client to differ.